agriculture_shop/static/src/js/my_tour.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { markup } from "@odoo/owl";
+
+registry.category("web_tour.tours").add("knowledge_tour", {
+  url: "/web",
```

Comment:
I think it changed before 17.4, so /web will be deprecated soon
```suggestion
  url: "/odoo",
```